addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/Config.java

```diff
@@ -37,13 +38,16 @@
     public static final String COMETVISU_ICON_ENABLE_MAPPING_PROPERTY = "enableMapping";
 
     public static final String COMETVISU_WEBFOLDER_PROPERTY = "webFolder";
+    public static final String COMETVISU_AUTODOWNLOAD_PROPERTY = "autoDownload";
 
-    public static String COMETVISU_WEBFOLDER = "runtime/web/";
+    public static String COMETVISU_WEBFOLDER = "/var/www/cometvisu";
```

Comment:
I think Kai did not yet respond on the location but we had such a question before, it should be located under *userdata* and there should be a method to retrieve the location.

@kaikreuzer correct me if I'm wrong and if you know the method for the location please mention 😄